multi-model-service/docs/code/access.md

```diff
@@ -0,0 +1,271 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/).
```

Comment:
Replaced the url here: https://github.com/redhat-ai-dev/model-catalog/pull/3/commits/5a9c53c0db72b919d974ed916ef14d74d5b96935

I know in the other PR for model techdocs you wanted in comments the source, I know in the md files right now there is the source that points to the url, do we want that dummied as well? or remove the source comment altogether since dummied wouldn't help anyone?

Reply:
I think that's fine